doc/sphinx/methods.rst

```diff
@@ -144,6 +144,66 @@ The LPA class available from cclib.method performs Löwdin population analysis a
     m = LPA(d)
     m.calculate()
 
+.. index::
+    single: methods; Bickelhaupt Population Analysis
+
+Bickelhaupt Population Analysis
+-------------------------------
+
+The Bickelhaupt class available from cclib.method performs Bickelhaupt population analysis that has been proposed in *Organometallics* 1996, 15, 13, 2923–2931. `doi:10.1021/om950966x <https://pubs.acs.org/doi/abs/10.1021/om950966x>`_
```

Comment:
alternatively the link could point to [https://doi.org/10.1021/om950966x](https://doi.org/10.1021/om950966x). Doesn't need to change just a tip if you come across a DOI with no link in the future